File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    //initialize n as integers
    int n;

    //prompt the user to input the height of bricks
    do
    {
        n = get_int("Height: ");
    }
    while (n < 0 || n > 23);

    //initialize i, j, k
    //i for the height
    //j for the space
    //k for the bricks #
    int i;
    int j;
    int k;

    //starting for loop counting the heights
    for (i = 0; i < n; i++)
    {
        //on the same line, print sufficient amount of space
        for (j = 0; j < n - i - 1; j++)
        {
            printf(" ");
        }
        //on the same line, print sufficient amount of bricks#
        for (k = 0; k < i + 2; k++)
        {
            printf("#");
        }
        //going to the next line
        printf("\n");
    }
}

File: pset6/mario/mario.py
from cs50 import get_int

while True:
    # prompt the user to input the height of bricks
    n = get_int("Height: ")
    if n >= 0 and n <= 23:
        break

# starting for loop counting the heights
for i in range(n):
    for j in range(n - i - 1):
        print(" ", end='')
    for k in range(i + 2):
        print("#", end='')
    print()
